template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/CmTemplateValidatorTest.java

```diff
@@ -353,6 +360,44 @@ public void testValidationIfNodeManagerCountWillBeHigherThanZeroInTheSameGroup()
                 Optional.of(clouderaManagerRepo), hostGroup, -2, Set.of(compute, worker));
     }
 
+    @Test
+    public void checkKafkaBrokerBlackListedUpscaleShouldBeAllowed() {
```

Comment:
I would simplify it with a parameterized test, also if we have a cutover on the version for a feature, I would test 1 or 2 versions less, the cutover version, and 1 or 2 versions more.

Reply:
I've provided a test for both cases allowed/disallowed. This logic has only 2 possible outcomes so I feel like the parametrised test is an overkill and makes the test less readable. 
Regardless this is wolf's domain, so I'll make the changes.